Formula/gaudi.rb

```diff
@@ -0,0 +1,62 @@
+class Gaudi < Formula
+  desc "Provides necessary interfaces and services for building HEP exp frameworks"
+  homepage "https://gaudi.web.cern.ch/gaudi/"
+  url "https://gitlab.cern.ch/gaudi/Gaudi/-/archive/v35r0/Gaudi-v35r0.tar.gz"
+  sha256 "c01b822f9592a7bf875b9997cbeb3c94dea97cb13d523c12649dbbf5d69b5fa6"
+  license "Apache-2.0"
+  head "ssh://git@gitlab.cern.ch:7999/gaudi/Gaudi.git"
+
+  depends_on "cmake" => :build
+  depends_on "aida-header"
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "clhep"
+  depends_on "cppunit"
+  depends_on "doxygen"
+  depends_on "gperftools"
+  depends_on "heppdt2"
+  depends_on "jemalloc"
+  depends_on "python"
+  depends_on "root"
+  depends_on "tbb"
+  depends_on "xerces-c"
+
+  def install
+    # Fix cmake issue with Foundation framework
+    text = File.read("GaudiKernel/CMakeLists.txt")
+    replace = text.gsub(/target_link_libraries\(GaudiKernel PUBLIC Foundation\)/,
+                        "target_link_libraries(GaudiKernel PUBLIC ${Foundation_FRAMEWORK})")
+    File.open("GaudiKernel/CMakeLists.txt", "w") { |file| file.puts replace }
+
+    system "cmake", ".", "-DBoost_NO_BOOST_CMAKE=ON", "-DGAUDI_USE_UNWIND=OFF", *std_cmake_args
+    system "make", "-j4"
+
+    # Remove the .db extension otherwise, the installation will crash
+    File.rename("Gaudi.confdb2.db", "Gaudi.confdb2")
```

Comment:
Has this been reported upstream?

Reply:
I sent an email about this issue

Reply:
They also have an issue tracker here where it would be better to report to: https://gitlab.cern.ch/gaudi/Gaudi/-/issues

This way you can leave a comment that references the issue, and anyone else who tries to update this formula will know if this fix is needed or not.

Reply:
I don't have the credentials to open an issue. I asked to a person that has it to do it for me (he is in the UK).